### PR TITLE
Add config flow to Meteo-France

### PR DESCRIPTION
**Pull request by Quentame**
**Description:**

Follows home-assistant/home-assistant#29927

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#29927

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/meteo_france.markdown [hunk 1]
```diff
@@ -5,21 +5,32 @@ logo: meteo-france.png
...
+
+There is two ways to integrate Météo-France in Home Assistant
+
+### Via the frontend
+
+Menu: *Configuration* -> *Integrations*
+
+Search for "Météo-France", add your city, click submit, you are done !
+
+Notice: all `monitored_conditions` will be added, see more details below.
```
Comment (frenck) [inline source/_integrations/meteo_france.markdown]: Not a real documentation review comment; We should remove the `monitored_conditions` altogether.

File: source/_integrations/meteo_france.markdown [hunk 2]
```diff
@@ -5,21 +5,32 @@ logo: meteo-france.png
 ha_release: 0.89
 ha_iot_class: Cloud Polling
```
Comment (frenck) [inline source/_integrations/meteo_france.markdown:6]: Missing `ha_config_flow` in the frontmatter.

File: source/_integrations/meteo_france.markdown [hunk 3]
```diff
@@ -5,21 +5,32 @@ logo: meteo-france.png
...
 
 - Sensor
 - Weather
 
 It displays the current weather along with a 4 days forecast and can create sensors based on the `monitored_conditions` set in your `configuration.yaml` file, including weather alerts from [Vigilance Météo-France](https://vigilance.meteofrance.com/)
 
-## Configuration
+## Setup the integration
+
+There is two ways to integrate Météo-France in Home Assistant
```
Comment (frenck) [inline source/_integrations/meteo_france.markdown]:
```suggestion
There is two ways to integrate Météo-France in Home Assistant.
```

File: source/_integrations/meteo_france.markdown [hunk 4]
```diff
@@ -5,21 +5,32 @@ logo: meteo-france.png
...
+
+### Via the frontend
+
+Menu: *Configuration* -> *Integrations*
+
+Search for "Météo-France", add your city, click submit, you are done !
```
Comment (frenck) [inline source/_integrations/meteo_france.markdown]:
```suggestion
Search for "Météo-France", add your city, click submit, you are done!
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/meteo_france.markdown (hunk 1 above)
Comment (Quentame) [reply source/_integrations/meteo_france.markdown]: Emmm, wait.

Should I remove this line or `monitored_conditionsˋ should be removed from the parent ?

Comment (frenck) [reply source/_integrations/meteo_france.markdown]: Ideally, it should be removed from the integration, for more information see: <https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md>

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/meteo_france.markdown (hunk 1 above)
Comment (Quentame) [reply source/_integrations/meteo_france.markdown]: Will look at it.

Thanks for the info !

**Comment by Quentame** [comment]
Can you remove the "doc-missing" tag on the parent PR ?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by frenck** [comment]
Done! Sorry, I missed that!

File: source/_integrations/meteo_france.markdown [hunk 5]
```diff
@@ -5,21 +5,31 @@ logo: meteo-france.png
...
 
 - Sensor
 - Weather
 
 It displays the current weather along with a 4 days forecast and can create sensors based on the `monitored_conditions` set in your `configuration.yaml` file, including weather alerts from [Vigilance Météo-France](https://vigilance.meteofrance.com/)
 
-## Configuration
+## Setup the integration
+
+There is two ways to integrate Météo-France in Home Assistant.
```
Comment (tjorim) [inline source/_integrations/meteo_france.markdown]:
```suggestion
There are two ways to integrate Météo-France in Home Assistant.
```

**Review by tjorim — CHANGES_REQUESTED** [review]

**Comment by Quentame** [comment]
Thanks @tjorim, also, if you have some time to review the parent PR, I'll appreciate!

**Review by frenck — DISMISSED** [review]

**Comment by Quentame** [comment]
Rebased to trigger the new pipeline.

**Review by frenck — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]

**Comment by Quentame** [comment]
Sorry to dismiss again @frenck ... doing parent modifications 😅

**Review by frenck — APPROVED** [review]

**Pull request merged**